lang/zh-t.xml

```diff
@@ -356,94 +356,94 @@
 		<completed_setup>已完成初期設定。
 請盡情享受Vita3K。</completed_setup>
 		<select_language>Select a language.</select_language>
-		<install_firmware>安裝固件。</install_firmware>
+		<install_firmware>安裝韌體。</install_firmware>
 		<install_highly_recommended>Installing both firmware files is highly recommended.</install_highly_recommended>
 		<installed>已安裝：</installed>
-		<download_firmware>下載固件</download_firmware>
-		<download_font_package>下載字庫包</download_font_package>
-		<install_firmware_file>安裝固件文件</install_firmware_file>
+		<download_firmware>下載韌體</download_firmware>
+		<download_font_package>下載字型套件</download_font_package>
+		<install_firmware_file>安裝韌體檔案</install_firmware_file>
 		<completed>已完成。</completed>
 		<next>Next</next>
 	</initial_setup>
 
 	<install_dialog>
 		<firmware_install>
-			<firmware_installing>Installation in progress, please wait...</firmware_installing>
-			<successed_install_firmware>固件已成功安裝。</successed_install_firmware>
-			<firmware_version>固件版本：</firmware_version>
-			<no_font_exist>固件字庫包不存在。
-請下載安裝固件字庫包。</no_font_exist>
-			<download_firmware_font_package>下載字庫固件包</download_firmware_font_package>
-			<firmware_font_package_note>Firmware font package is needed for some applications and also for Asian regional font support. (Generally Recommended)</firmware_font_package_note>
-			<delete_firmware>Delete the firmware installation file?</delete_firmware>
+			<firmware_installing>安裝進行中，請稍候…</firmware_installing>
+			<successed_install_firmware>韌體已成功安裝。</successed_install_firmware>
+			<firmware_version>韌體版本：</firmware_version>
+			<no_font_exist>韌體字型套件不存在。
+請下載並安裝韌體字型套件。</no_font_exist>
+			<download_firmware_font_package>下載韌體字型套件</download_firmware_font_package>
+			<firmware_font_package_note>某些應用程式和亞洲區域字型支援需要韌體字型套件（一般建議安裝）。</firmware_font_package_note>
```

Comment:
no need change, use english, if no install firmware, here not display the normal text.